packages/appstore/src/components/cfd-account-type-details/cfd-account-type-details.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import cfdAccountDetails from 'Constants/cfd-account-type-details';
+
+type TCFDAccountTypeDetails = {
+    platform: 'mt5' | 'dxtrade';
+    is_eu: boolean;
+    account_type: 'synthetic' | 'financial' | 'financial_stp';
+};
+
+const CFDAccountTypeDetails = ({ platform, is_eu, account_type }: TCFDAccountTypeDetails) => {
+    const [cfd_account_details, setCfdAccountDetails] = React.useState({
+        title: '',
+        description: '',
+        leverage: { key: () => '', value: () => '' },
+        'margin-call': { key: () => '', value: () => '' },
+        'stop-out-level': { key: () => '', value: () => '' },
+        'number-of-assets': { key: () => '', value: () => '' },
```

Comment:
we don't have to set initial values. If `cfd_account_details` is undefined, then we can return null in the render